tests/components/snoo/test_init.py

```diff
@@ -0,0 +1,11 @@
+"""Test init for Snoo."""
+
+from homeassistant.core import HomeAssistant
+
+from . import async_init_integration
+
+
+async def test_async_setup_entry(hass: HomeAssistant, bypass_api) -> None:
+    """Test a successful setup entry."""
+    await async_init_integration(hass)
+    assert len(hass.states.async_all("sensor")) == 2
```

Comment:
We can also test that the config entry we had has `.state is ConfigEntryState.LOADED`